frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -470,9 +470,17 @@ export class OverviewPageComponent extends React.Component<OverviewProps, State>
   }
 
   fetchTLS(isAscending: boolean, sortField: SortField<NamespaceInfo>): void {
-    _.chunk(this.state.namespaces, 10).forEach(chunk => {
+    const uniqueClusters = new Set<string>();
+
+    this.state.namespaces.forEach(namespace => {
+      if (namespace.cluster) {
+        uniqueClusters.add(namespace.cluster);
+      }
+    });
```

Comment:
Does this work in multi-cluster mode? Unfortunately depending on where the namespaces come from in `this.state.namespaces` sometimes you have the cluster in `namespace.cluster` and sometimes you don't. This is partially because the namespace names can come from the URL and we lose the cluster when that happens.

Reply:
yes it works with multi/single clusters, also added a filter here and in validations request to send namespaces for that cluster only, otherwise all namespaces were sent comma separated.
![Screenshot from 2024-02-12 12-17-36](https://github.com/kiali/kiali/assets/604313/bec9a07b-7bf8-442e-b49b-9160eae2ed90)
![Screenshot from 2024-02-12 12-17-16](https://github.com/kiali/kiali/assets/604313/bae0f090-8205-4b29-8570-39d70d0dea8a)